tests/accumulator_test.py

```diff
@@ -2,190 +2,219 @@
-    self.count += 1
-    return self
+    def add_value(self, v):
+        self.sum += v
+        self.count += 1
+        return self
 
-  def add_accumulator(self,
-                      accumulator: 'MeanAccumulator') -> 'MeanAccumulator':
-    if not isinstance(accumulator, MeanAccumulator):
-      raise TypeError("The accumulator to be added is not of the same type.")
-    self.sum += accumulator.sum
-    self.count += accumulator.count
-    return self
+    def add_accumulator(self,
+                        accumulator: 'MeanAccumulator') -> 'MeanAccumulator':
+        if not isinstance(accumulator, MeanAccumulator):
+            raise TypeError(
+                "The accumulator to be added is not of the same type.")
+        self.sum += accumulator.sum
+        self.count += accumulator.count
+        return self
 
-  def compute_metrics(self):
-    if self.count == 0:
-      return float('NaN')
-    return self.sum / self.count
+    def compute_metrics(self):
+        if self.count == 0:
+            return float('NaN')
+        return self.sum / self.count
 
 
 # Accumulator classes for testing
-class SumOfSquaresAccumulator(Accumulator):
+class SumOfSquaresAccumulator(accumulator.Accumulator):
 
-  def __init__(self, accumulators: typing.Iterable[
-    'SumOfSquaresAccumulator'] = None):
-    self.sum_squares = np.sum([concat_acc.sum_squares
-                               for concat_acc in
-                               accumulators]) if accumulators else 0
+    def __init__(
+            self,
+            accumulators: typing.Iterable['SumOfSquaresAccumulator'] = None):
+        self.sum_squares = np.sum([
+            concat_acc.sum_squares for concat_acc in accumulators
+        ]) if accumulators else 0
 
-  def add_value(self, v):
-    self.sum_squares += v * v
-    return self
+    def add_value(self, v):
+        self.sum_squares += v * v
+        return self
 
-  def add_accumulator(self,
-                      accumulator: 'SumOfSquaresAccumulator') -> 'SumOfSquaresAccumulator':
-    if not isinstance(accumulator, SumOfSquaresAccumulator):
-      raise TypeError("The accumulator to be added is not of the same type.")
-    self.sum_squares += accumulator.sum_squares
-    return self
+    def add_accumulator(
+            self, accumulator: 'SumOfSquaresAccumulator'
+    ) -> 'SumOfSquaresAccumulator':
+        if not isinstance(accumulator, SumOfSquaresAccumulator):
+            raise TypeError(
+                "The accumulator to be added is not of the same type.")
+        self.sum_squares += accumulator.sum_squares
+        return self
 
-  def compute_metrics(self):
-    return self.sum_squares
+    def compute_metrics(self):
+        return self.sum_squares
+
+
+class CountAccumulatorTest(unittest.TestCase):
+
+    def test_without_noise(self):
+        count_accumulator = accumulator.CountAccumulator(
+            accumulator.CountParams(), list(range(5)))
+        assert count_accumulator.compute_metrics() == 5
```

Comment:
Nit: assert -> self.assertEqual  (and below)

Explanation: it's better to use asserts from unittest framework, since it allows better failure management

Reply:
Thx, done.